src/App.jsx

```diff
@@ -1,23 +1,38 @@
 import * as React from 'react'
 import Card from './components/Card/Card'
 import Header from './components/Header/Header'
-
+import questions from './data.json'
 
 export default () => {
   return (
     <div>
-      <Header title="Cards with React" />
+      <Header title="Quiz-App with React" />
 
       <section className="Card-list">
-        <Card heading="I'm a card" text="Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ratione, illum! Cupiditate quibusdam consequuntur est voluptatibus velit? Praesentium quibusdam facere non!" />
-
-        <Card isBookmarked heading="I'm a bookmarked Card" text="Lorem ipsum dolor sit amet consectetur, adipisicing elit. Ratione, illum! Cupiditate quibusdam consequuntur est voluptatibus velit? Praesentium quibusdam facere non!" />
+        {questions.map(element => {
+          const {
+            id,
+            question,
+            correct_answer,
+            category,
+            difficulty,
+            isBookmarked,
+            isAnswerVisible,
+          } = element
```

Comment:
You can write this a bit shorter and directly destructure it from the element:
```suggestion
        {questions.map(({ id,
            question,
            correct_answer,
            category,
            difficulty,
            isBookmarked,
            isAnswerVisible }) => {
```